custom/icds_reports/static/js/directives/month-filter/month-filter.directive.js

```diff
@@ -115,7 +119,14 @@ function MonthFilterController($scope, $location, $uibModal, storageService) {
             controllerAs: '$ctrl',
             resolve: {
             },
-        });
+        };
+
+        if (vm.isSSD && !vm.isSSDAvailableForGivenDate) {
+            modalOptions['backdrop'] = 'static';
+            modalOptions['keyboard'] = false;
```

Comment:
I think could be worth a comment explaining why we're doing this (basically what you said in the PR description) since is quite non-obvious.